src/v2/Artsy/Relay/createRelaySSREnvironment.ts

```diff
@@ -98,6 +99,7 @@ export function createRelaySSREnvironment(config: Config = {}) {
       size: 100, // max 100 requests
       ttl: 900000, // 15 minutes
       clearOnMutation: true,
+      disableServerSideCache: !!user, // disable server-side cache if logged in
```

Comment:
Note that this compliments individual route configs. For example, we never cache artworks (yet 😄): 

https://github.com/artsy/force/blob/7544c3e3801efbee96be8c65c4685efe90faff9e/src/v2/Apps/Artwork/routes.tsx#L23-L25